src/azul/__init__.py

```diff
@@ -1226,6 +1226,14 @@ class GitStatus(TypedDict):
     def _git_status_env(self) -> dict[str, str]:
         return {'azul_git_' + k: str(v) for k, v in self.git_status.items()}
 
+    @property
+    def _deployment_env(self) -> dict[str, str]:
```

Comment:
Name is a bit general. Expecting that we might need to do this for other variables:

```suggestion
    def _explicitly_exported_env(self) -> dict[str, str]:
```